app/services/export_service/order_export.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module ExportService
```

Comment:
it should be called without `Service`. IMO it's not a good practice

Reply:
I have fixed it.